README.md

```diff
@@ -15,4 +15,124 @@ As a user, I would like to directly customize my feature using inline editing fo
+
+1. Create a new file called `.env` at the root of your project directory. Inside this file, you'll need to add in the content base and access token for your API
+
+2. Go to your [orgs homepage](https://redirector.arcpublishing.com/home/) or if you don't have one, use our [demo organization](https://demo.arcpublishing.com/home/)
+
+3. Click developer center and click "create read-only token"
+
+4. Enter your email address into the description field and click create. This will create your access token
+
+5. Copy the content at the bottom where it says "Copy the following into your PB .env file if setting up PageBuilder" and place it into your .env. It should be something like this:
+```
+CONTENT_BASE=https://api.sandbox.{org}.arcpublishing.com 
+ARC_ACCESS_TOKEN={token_here}
+```
+where `org` is the name of your organization
+
+6. Now that we have access to our orgs API, we need to create a content source to retrieve the API data. All content sources goes into the `content/sources` folder
+
+7. Open the `content/sources` folder and create a new file called `content-api.js`
+
+8. Give you content source a schema name. For example:
+```
+const schemaName = 'article';
+```
+
+9. Are there params that need to be provided to use this content source? Then add them in:
+```
+const params = {
+  website_url : 'text',
+  published: 'text',
+};
+```
+In our example above, this content source is expecting a `website_url` and `published` date
+
+10. We need to define the URI which the content source will call. Create a resolve function which takes in a `key` holding the params defined above and returns the URI
+```
+const resolve = (key) => {
+  const requestUri = `/content/v4/stories/?website_url=${ key.website_url || key }&website={org}`;
+
+  return (key.hasOwnProperty('published')) ? `${requestUri}&published=${key.published}` : requestUri
+}
+```
+
+11. *(This is only for those using demo as the org)* If you are using `demo` as your org then you probably don't have any data available. As a work-around, go to fusion-08 and download the `helpers/sample-content.js` file into a `helpers` folder in the root of your project. Now, we can import this sample data into our content source and return it when this content source is called like so:
+
+```
+import { CONTENT } from '../../helpers/sample-content';
+// ... other functions / consts
+const fetch = (query = {}) => {
+  return CONTENT;
+};
+```
+
+12. Finally, export your content source:
+```
+export default {
+  resolve,
+  fetch,  // you only need this if using the sample data mentioned on step #11
+  params,
+  schemaName,
+}
+```
+
+13. Since content sources usually return a lot of data, we need to define what fields we want in order to reduce the size and improve performance. This is done by creating a schema for our content source.
+
+14. From root, open the `content/schemas` folder - this is where all schemas reside
```

Comment:
Since schemas are now ignored on content sources, I think we should skip these steps for adding the schema. https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/using-graphql-schema.md

Reply:
Schemas aren't necessary for returning data, however, they are used when grouping data sources together in feature level content configuration later. Also, in our content source deep dive from Arc Week, the Engine team still recommended having a schema as best practices since they hope to integrate with them in the future.

Reply:
Thanks for clarifying! I was a little confused by the note in the documentation, and thought it would cause confusion for anyone being introduced to it. Sounds like it's still good for people to be introduced to it, but I wonder if including the link to the documentation with some context about how they are used and might be in the future might be worthwhile.

Reply:
Agreed! Definitely "not necessary, but recommended as best practice" 👍